models/category.js

```diff
@@ -0,0 +1,50 @@
+const { query } = require('../db/index');
+
+exports.all = async () => {
+  const categories = (await query(
+    'SELECT * FROM "categories"',
+  )).rows;
+  return categories;
+};
+
+exports.create = async properties => {
+  const errors = await validate(properties);
+  if (errors) {
+    return { errors };
```

Comment:
do we want to return the malformed object as well?

Reply:
done

Reply:
Yes, in other models. Category can be done when we are CRUDing the resource in ...5...4...3...